docs/src/modules/components/PickersPlayground.tsx

```diff
@@ -32,14 +32,11 @@ import {
 } from '@mui/x-date-pickers/StaticDateTimePicker';
 import { DateOrTimeView } from '@mui/x-date-pickers/models';
 import { PickersShortcutsItem } from '@mui/x-date-pickers/PickersShortcuts';
-import { DateRange } from '@mui/x-date-pickers-pro/internals/models';
+import { DateRange } from '@mui/x-date-pickers-pro';
```

Comment:
I need to move this interface to `@mui/x-date-pickers-pro/models` in a follow up, it's a shame to have to use the root import here but it's the best we can do for now

Reply:
Were you considering doing a follow-up?
I can do it as well. 👍

Reply:
Yes that's something I wanted to do for some time :+1: 
I'll handle it next week